lib/data/solr_fields_config.yml

```diff
@@ -404,6 +404,10 @@ series_issn:
   type: num
 series_other_ids:
   type: e
+sersol_number:
+  type: e
+  attr:
+    stored 
```

Comment:
Sorry I missed this the first time around. Should be:

` - stored`

Reply:
Done!